data/avro-schema/build.gradle

```diff
@@ -63,9 +63,14 @@ def generateOSGiPackageInfo = tasks.register("generateOSGiPackageInfo", Task) {
             if (!it.name.contains("package-info") && it.name.endsWith('.java')) {
                 def packageName = ((it.text =~ "package (.+);")[0][1])
                 packages << packageName
-                def classCapture = it.text.normalize() =~ "@org.apache.avro.specific.AvroGenerated\npublic class (\\w+) extends org.apache.avro.specific.SpecificRecordBase"
-                if (classCapture.any()) {
-                    classes << packageName + '.' + classCapture[0][1]
+                def normalizedText = it.text.normalize()
```

Comment:
It's probably worth explaining in the PR description why this change is needed?

Reply:
Yep, done.